src/freenet/node/NodeClientCore.java

```diff
@@ -1980,7 +1980,7 @@ public String getMyName() {
 	public FilterCallback createFilterCallback(URI uri, FoundURICallback cb) {
 		if(logMINOR)
 			Logger.minor(this, "Creating filter callback: " + uri + ", " + cb);
-		return new GenericReadFilterCallback(uri, cb,null, toadletContainer);
+		return new GenericReadFilterCallback(uri, cb, toadletContainer);
```

Comment:
I would suggest reverting this and instead getting rid of that additional constructor because I don’t really see a pressing need for it… I believe an additional constructor will cause more confusion than a `null` parameter.

Reply:
you’re right — a single-use constructor doesn’t really help here.